pkg/cmd/login/login.go

```diff
@@ -124,7 +124,7 @@ func GetDefaultAuthConfig(checkCredStore bool, serverAddress string, isDefaultRe
 			return nil, err
 		}
 	}
-	var authconfig = dockercliconfigtypes.AuthConfig{}
+	authconfig := dockercliconfigtypes.AuthConfig{}
```

Comment:
Probably these cosmetic changes should be a separate commit

Reply:
is it the gofumpt